content/fr/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -0,0 +1,1225 @@
+---
+title: Déploiements
+feature:
+  title: Déploiements et restaurations automatisés
+  description: >
+    Kubernetes déploie progressivement les modifications apportées à votre application ou à sa configuration, tout en surveillant l'intégrité de l'application pour vous assurer qu'elle ne tue pas toutes vos instances en même temps.
+    En cas de problème, Kubernetes annulera le changement pour vous.
+    Profitez d'un écosystème croissant de solutions de déploiement.
+
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Un _Déploiement_ (Deployment en anglais) fournit des mises à jour déclaratives pour [Pods](/docs/concepts/workloads/pods/pod/) et [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/).
+
+Vous décrivez un _état désiré_ dans un déploiement et le déploiement {{< glossary_tooltip term_id="controller" >}} change l'état réel à l'état souhaité à un rythme contrôlé.
+Vous pouvez définir des déploiements pour créer de nouveaux ReplicaSets, ou pour supprimer des déploiements existants et adopter toutes leurs ressources avec de nouveaux déploiements.
+
+{{< note >}}
+Ne gérez pas les ReplicaSets appartenant à un déploiement.
+Pensez à ouvrir un ticket dans le dépot Kubernetes principal si votre cas d'utilisation n'est pas traité ci-dessous.
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Cas d'utilisation
+
+Voici des cas d'utilisation typiques pour les déploiements:
+
+* [Créer un déploiement pour déployer un ReplicaSet](#creating-a-deployment).
+  Le ReplicaSet crée des pods en arrière-plan.
+  Vérifiez l'état du déploiement pour voir s'il réussit ou non.
+* [Déclarez le nouvel état des Pods](#updating-a-deployment) en mettant à jour le PodTemplateSpec du déploiement.
+  Un nouveau ReplicaSet est créé et le déploiement gère le déplacement des pods de l'ancien ReplicaSet vers le nouveau à un rythme contrôlé.
+  Chaque nouveau ReplicaSet met à jour la révision du déploiement.
+* [Revenir à une révision de déploiement antérieure](#rolling-back-a-deployment) si l'état actuel du déploiement n'est pas stable.
+  Chaque restauration met à jour la révision du déploiement.
+* [Augmentez le déploiement pour traiter plus de charge](#scaling-a-deployment).
+* [Suspendre le déploiement](#pausing-and-resuming-a-deployment) d'appliquer plusieurs correctifs à son PodTemplateSpec, puis de le reprendre pour démarrer un nouveau déploiement.
+* [Utiliser l'état du déploiement](#deployment-status) comme indicateur qu'un déploiement est bloqué.
+* [Nettoyer les anciens ReplicaSets](#clean-up-policy) dont vous n'avez plus besoin.
```

Comment:
The fragment identifiers in these hyperlinks don't match the headings and their automatic fragment identifiers.